clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/validators/StandardGceAttributeValidator.groovy

```diff
@@ -173,15 +173,15 @@ class StandardGceAttributeValidator {
 
   def validateRegion(String region, GoogleNamedAccountCredentials credentials) {
     if (validateNotEmpty(region, "region") && credentials) {
-      if (!(region in credentials.regionToZonesMap)) {
+      if (!(region in credentials.regions.collect { it.name })) {
```

Comment:
We can use the groovy spread-dot operator here and below:

`credentials.regions*.name`